src/components/api.ts

```diff
@@ -126,6 +98,10 @@ export async function fetchRatingFromVivino(
     // Find the `data-preloaded-state` attribute
     const searchPageElement = $('#search-page')
     if (!searchPageElement.length) {
+      // TODO: need to figure out why we end up here sometimes
+      if ($('.wine-card__content').length >0) { 
+        console.log('Vivino: No preloaded state found, but wine cards exist.')
```

Comment:
Though it would be useful for debugging, I think the console.log has to be removed because the linter does not permit it and it would also expose the log to users. We might be able to create a global log function that logs if we are in development. Like we did with the websocket permission